src/blog/2022/09/static-ips.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Introducing Static IP Addresses on FlowForge Cloud #9
+subtitle: Static IP addresses are here for your FlowForge Cloud projects’ outbound connections
+description: Static IP addresses are here for your FlowForge Cloud projects’ outbound connections
+date: 2022-09-26
+authors: ["rob-marcer"]
+---
+
+On Friday last week we updated FlowForge Cloud to use static IP addresses for outbound traffic. This will allow you to predict which IP addresses your traffic will come from for example when traversing a firewall or accessing a remote database.
+
+You will need to manually suspend then start each of your projects (a restart will not move your projects to the fixed IP addresses). Once that action is completed all outbound connections will come from one of our static IP addresses.
+
+Any inbound traffic should still use the hostname assigned to each of your projects, you cannot use our IP addresses to route http traffic to your projects.
+
+You can view our IP addresses in the [Docs](https://flowforge.com/docs/cloud/#ip-addresses) section of our website.
```

Comment:
This link won't work until the new version of the docs is published with the 0.10 release, we could use https://github.com/flowforge/flowforge/tree/main/docs/cloud#ip-addresses and then update the blog post after 0.10 is live

Reply:
@sammachin updated to that link for now and changed all mentions of IP addresses to be singular.